src/Core/Grid/Column/ColumnCollection.php

```diff
@@ -52,31 +51,19 @@ public function add(ColumnInterface $column)
      */
     public function addAfter($id, ColumnInterface $newColumn)
     {
-        if (!isset($this->items[$id])) {
-            throw new ColumnNotFoundException(sprintf(
-                'Column with id "%s" was not found.', $id
-            ));
-        }
+        $this->insertByPosition($id, $newColumn, 'after');
```

Comment:
After and before can be const :)

Reply:
i would like to make them private constants, as they are not meant to be used outside this class, but it must support PHP 5.6. :/

Reply:
If you don't want to use public const and allow only to value in `insertByPosition` you can use a boolean instead =)

Reply:
disagree on boolean solution, we lose context :)